docs/reference/elasticsearch/mapping-reference/mapping-source-field.md

```diff
@@ -308,18 +308,19 @@ PUT my-index-000001
 }
 ```
 
-::::{admonition} Think before disabling the _source field
+::::{admonition} Think before you disable the _source field!
```

Comment:
```suggestion
::::{admonition} Disabling the `_source` field
```

This is a nit, but I think it's more in keeping with your edits, to make the text clearer,  more concise, and more professional.

 Also, we try to avoid exclamation marks in prose as much as possible. :)